animations/lib/src/misc/focus_image.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+
+class FocusImageDemo extends StatelessWidget {
+  static String routeName = '/misc/focus_image';
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('Focus Image')),
+      body: Grid(),
+    );
+  }
+}
+
+class Grid extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: GridView.builder(
+        itemCount: 40,
+        gridDelegate:
+        SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 4),
```

Comment:
Clearly a good WotW candidate. 😆

Reply:
@efortuna Now that I'm thinking about it, though, we should totally do some obscure widget like this for April Fool's next year.